webpack.config.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+
+function getEntries(dir) {
+  return Object.assign.apply({},
+    fs.readdirSync(dir, {
+      withFileTypes: true
+    })
+    .filter(dirent => dirent.isDirectory())
```

Comment:
dirent is a really weird variable name, what's this mean?

Reply:
https://nodejs.org/api/fs.html#class-fsdirent